robot_skills/src/robot_skills/robot_part.py

```diff
@@ -32,6 +33,15 @@ def __init__(self, robot_name, tf_listener):
         # If no such check exists, then assume it's operational unless overridden in subclass
         self._operational = True
 
+    @property
+    def id(self):
```

Comment:
This is something to look out with: `id` is also a Python thing itself, that is why it's called `_id` in the constructor. Any Python object already has an `id` but that is a built-in thing we do now want to mess with. An option is to name the method/property `id_`